tests/test_smtp/test_smtp_util.py

```diff
@@ -0,0 +1,43 @@
+"""Tests relating to the module smtp.smtp_util."""
+
+import unittest
+from io import BytesIO
+
+from itk_dev_shared_components.smtp import smtp_util
+from itk_dev_shared_components.smtp.smtp_util import EmailAttachment
+
+
+class TestTreeUtil(unittest.TestCase):
```

Comment:
It is meant to be run interactively since I don't have another good way to do assertions for this test other than checking the resulting emails manually. 

I could use the Graph part of the library to automatically check the emails butt this does introduce a problem regarding delivery times, scanning of files etc